### ISSUE TITLE
podSpec missing

### ISSUE THREAD

**Issue by Oggerschummer** [Seoras/ThirdPartyMailer#3]
Hi,

thanks for porting this to obj-c.
Could you add a podspec so it can directly be used with cocoapods ?

Thanks!

**Comment by Seoras** [comment]
Can cocoapods support making changes to the app’s info.plist? I don’t think it can although I may be wrong.
That is why I haven’t added pod support. The url support required for this utility are a key part of it. See the readme file.

**Comment by Oggerschummer** [comment]
No, it does not, neither does it write the app code to actually implement it. That's  part of what I call "configuration and implementation".

But I prefer downloading code as a pod, manual installation and updating is additional effort.
The original swift version does use a podspec for that reason.
But as it's your baby you decide on it.

**Comment by Seoras** [comment]
Try this in your pod file. I've tested it on an App and it appears to work fine for me.

`pod 'ThirdPartyMailerObjC', :git => 'https://github.com/Seoras/ThirdPartyMailer.git'`

If it works for you I'll commit it to the pod trunk

**Comment by Oggerschummer** [comment]
Hi,
sorry for the late response, but I’ve been busy at the customer side for some days.
Just changed the pod file as  you proposed and the pod did install properly.

It complies, lets ship it ;-)

Thomas
